README.md

```diff
@@ -175,6 +175,7 @@ A curated list of amazingly awesome awesomeness.
 	- [Docker](https://github.com/veggiemonk/awesome-docker)
 	- [Documentation](https://github.com/PharkMillups/beautiful-docs)
 	- [Dotfiles](https://github.com/webpro/awesome-dotfiles)
+	- [E-commerce](https://github.com/snipcart)
```

Comment:
The link goes to Snipcart's Github organization profile and not an e-commerce awesome repo.